index.js

```diff
@@ -130,10 +132,16 @@ module.exports = function(version) {
             var instructionObject;
             if (instructions[language][version].modes[mode]) {
                 instructionObject = instructions[language][version].modes[mode];
-            } else if (instructions[language][version][type][modifier]) {
-                instructionObject = instructions[language][version][type][modifier];
             } else {
-                instructionObject = instructions[language][version][type].default;
+              // omit side from off ramp if same as driving_side
+              // note: side will be undefined if the input is from OSRM <5.14
+              // but the condition should still evaluate properly regardless
```

Comment:
Weird that the linter complained about indentation below but not on these comment lines. 🤷‍♂️